app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -293,7 +294,15 @@ public void toggleLimitedConnectionMode() {
 
     public void centerMapToPlace(Place place) {
         setSelectedItemId(NavTab.NEARBY.code());
-        nearbyParentFragment.centerMapToPlace(place);
+//        addd the postDelayed function so that before call the nearbyParentFragment.centerMapToplace(). nearbyParentFragment
+//      initialise complete so that it so that it dooes give any Nullpointer exception when we nearbyParentFragment.centerMapToplace() is called
+      Handler handler=new Handler();
+      handler.postDelayed(new Runnable() {
+        @Override
+        public void run() {
+          nearbyParentFragment.centerMapToPlace(place);
+        }
+      },1000);
```

Comment:
This is not a good practice because we can never know it our time is enough or not. We should use a callback when the map is ready instead.